### PR TITLE
Added Voice Input In Search Bar

### PR DESCRIPTION
**Pull request by Achintya999**
Makes it easier to search for users.

### CODE REVIEW

**Comment by pocmo** [comment]
Thank you for your patch!

Can you please open an issue for this first? Usually we want to discuss new features like this with our UX and product team. :)

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 1]
```diff
@@ -245,6 +263,33 @@ public void onClick(View view) {
         }
     }
 
+    private  void startSpeechToText(){
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 2]
```diff
@@ -245,6 +263,33 @@ public void onClick(View view) {
...
+
+        Intent intent = new Intent(RecognizerIntent.ACTION_RECOGNIZE_SPEECH);
+        intent.putExtra(RecognizerIntent.EXTRA_LANGUAGE_MODEL, RecognizerIntent.LANGUAGE_MODEL_FREE_FORM);
+        intent.putExtra(RecognizerIntent.EXTRA_LANGUAGE, Locale.getDefault());
+        intent.putExtra(RecognizerIntent.EXTRA_PROMPT, "Speak The Site Where You Want To Go!");
+        try{
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: 'try' is not followed by whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 3]
```diff
@@ -245,6 +263,33 @@ public void onClick(View view) {
...
+            startActivityForResult(intent, REQ_CODE_SPEECH_INPUT);
+        }catch (ActivityNotFoundException e){
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: 'catch' is not preceded with whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\}' is not followed by whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java (hunk 1 above)
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java (hunk 2 above)
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: 'try' is not followed by whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java (hunk 3 above)
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: 'catch' is not preceded with whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\}' is not followed by whitespace\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

**Comment by Achintya999** [comment]
#1155

**Pull request closed without merging**